cdap-api/src/main/java/co/cask/cdap/api/DatasetConfigurer.java

```diff
@@ -32,13 +32,15 @@
    *
    * @param stream {@link Stream}
    */
+  @Deprecated
```

Comment:
It would be good to add a deprecation javadoc like the follows (over here and at all other places below) - 
```
/**
 * @deprecated  As of release 5.0, replaced by {@link replacement-link}
 */
```